### ISSUE TITLE
The "merges" folder is missing

### ISSUE THREAD

**Issue by valgen** [phonegap/phonegap-app-desktop#859]
## _System Configuration_
Windows 10

### PhoneGap Desktop version
v0.4.5

### PhoneGap Mobile App version
v1.8.2

### Mobile devices & Mobile OS version
Android
v6.0.1

### Computer OS version
v10

### Network Type (home/corporate)
home

### Is your network using a proxy?
no

### Computer / Network security (firewall / anti-virus / disk encryption)
Avira anti-virus

### Are you using VMs on your computer?
no

### Do you have multiple network adapters (or virtual network adapters)?
no

### Are you using VPN?
no

### Expected Behaviour
Should create a "merges" folder.

### Actual Behaviour
Does not create a "merges" folder.
Even if I add the "merges" folder manually, PhoneGap Desktop ignores it.

### Steps to Reproduce 
- Open PhoneGap Desktop
- Create new PhoneGap Project -> Hello World
- The "merges" folder is missing
- It doesn't work even if I add the "merges" folder manually and create an alternative "css/index.css" to overwrite the original one.

#### Error messages
none

**Comment by valgen** [comment]
Hi there,

Does anyone actually look into this issues?

**Comment by surajpindoria** [comment]
Hi @valgen,

Sorry for the delay, I saw this the other day but was wrapped up with some other tasks. The merges folder is not created by default when you create a new project so you will need to make the folder yourself. Also the Desktop App will not do anything with this folder as all it does is serve the content directly from your projects www directory. If you want to use the merges folder you will need to use the phonegap-cli and do a full build of your app.

**Comment by valgen** [comment]
Hi @surajpindoria,

Thank you for your reply.

I was already aware of the phonegap-cli full build but this doesn't solve the issue with PhoneGap Desktop.

Is there any plan to support the merges folder within PhoneGap Desktop?

Also I reckon this issue (#859) should be labelled also as **enhancement**.

This feature is really needed and I'm sure it will be beneficial for everyone in the PhoneGap community.

**Comment by surajpindoria** [comment]
Unfortunately there is no plan to support the merges folder since that would also require full builds within the Desktop App. This comes with many problems by itself as it would require users of the app to have the correct SDKs and tools installed to work, which is out of scope for what the Desktop App is trying to accomplish.

**Comment by valgen** [comment]
I'm sorry but I have to disagree with you.

I thought the scope of the Desktop App was to speed up the development process.

If a user has a project which vitally depends on the "merges" folder where he loads different CSS and JS based on the platform then he won't be able to test it within PhoneGap Desktop therefore this one becomes useless for the user.

I will not reply to this issue anymore but please label it appropriately and don't close it, so if anyone else has this same problem he will be able to understand why, without wasting his time.

**Comment by surajpindoria** [comment]
The desktop app was designed as a quick intro into the PhoneGap world without having to install all the SDKs and for people who are not comfortable using the CLI. Over time we have added new functionality, but even today it is only able to accomplish a small subset (create and serve) of the commands that the CLI can do. We looked into adding more commands but were facing many technical hurdles that went against this original goal. Therefore we have decided to leave it with the current commands.
